### PR TITLE
[Singleton] Add get_only_instance

### PR DESCRIPTION
**Pull request by valouvaliavlo**

### CODE REVIEW

File: octobot_commons/singleton/singleton_class.py [hunk 1]
```diff
@@ -17,3 +17,14 @@ def instance(cls, *args, **kwargs):
         if cls not in cls._instances:
             cls._instances[cls] = cls(*args, **kwargs)
         return cls._instances[cls]
+    
+    @classmethod
+    def get_only_instance(cls):
```
Comment (Herklos) [inline octobot_commons/singleton/singleton_class.py:22]: I would have named it `get_instance_if_exists()`

**Review by Herklos — COMMENTED** [review]
:+1:

File: octobot_commons/singleton/singleton_class.py (hunk 1 above)
Comment (valouvaliavlo) [reply octobot_commons/singleton/singleton_class.py]: done

**Review by Herklos — APPROVED** [review]
:+1:

File: octobot_commons/singleton/singleton_class.py [hunk 2]
```diff
@@ -17,3 +17,14 @@ def instance(cls, *args, **kwargs):
...
+    @classmethod
+    def get_instance_if_exists(cls):
+        """
+        Return the instance if it exist
+        Return the class instance if it exist
+        :return: the class only instance if it exist otherwise None
+        """
+        if cls in cls._instances:
+            return cls._instances[cls]
+        return None
```
Comment (Herklos) [inline octobot_commons/singleton/singleton_class.py:30]: Sorry i just realized that it should be more pythonic to do : 
```suggestion
        try:
            return cls._instances[cls]
        except KeyError:
            return None
```

Comment (valouvaliavlo) [reply octobot_commons/singleton/singleton_class.py:30]: 👍

**Comment by valouvaliavlo** [comment]
There is no copyright on SIngleton class, maybe add it ?

**Comment by Herklos** [comment]
> There is no copyright on SIngleton class, maybe add it ?

Yes please :)

**Review by Herklos — APPROVED** [review]
:+1:

**Review by GuillaumeDSM — APPROVED** [review]

**Review by GuillaumeDSM — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Pull request merged**